scripts/init.groovy.d/aa_configure_credentials.groovy

```diff
@@ -0,0 +1,370 @@
+// output init hook stage
+println '### JENKINS AUTOCONFIG: CREDENTIALS ###'
+
+/**
+  * Check if Jenkins should not configure itself and skip.
+  */
+
+Boolean skipConfiguringJenkins() {
+    !(System.env?.ENVIRONMENT in ['dev', 'staging']) ||
+        (new File("${Jenkins.instance.rootDir}/autoConfigComplete").exists())
```

Comment:
No action needed here, but just curious why `(new File("${Jenkins.instance.rootDir}/autoConfigComplete").exists())` is on a new line here, instead of after the `||` on the same line above?

Reply:
This was a snippet from Sam, i'm assuming so it can satisfy maximum line length for his style.

Reply:
That makes sense. I hadn't considered line length preferences